src/Components/RouteTabs.tsx

```diff
@@ -39,11 +39,7 @@ export const RouteTab: React.FC<BaseTabProps & RouterLinkProps> = ({
 RouteTab.displayName = "RouteTab"
 
 export const RouteTabs: React.FC<BaseTabsProps> = ({ children, ...rest }) => {
-  return (
-    <BaseTabs justifyContent="space-between" mx={[-4, 0]} px={[2, 0]} {...rest}>
```

Comment:
`justifyContent` got added this for a one-off use-case 2 years ago and hit areas on the settings page tabs have been broken since (!). Surprisingly hard to notice this sort of thing... another vote in the dramatically restrict Palette components types bucket.